### PR TITLE
LocalBearTestHelper: Add ``check_order`` in doc

### PR DESCRIPTION
**Pull request by pareksha**
This adds ``check_order`` parameter in docstring
of ``check_results`` which was previously missing.

Closes https://github.com/coala/coala/issues/5126

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by pareksha** [comment]
It's not necessary @prakashdanish.
Shortlog will exceed the character limit if i do so.

**Comment by newbazz** [comment]
@prakashdanish `.py` is understood as while importing we do `from x.y.x import w` an not `from x.y.z.py import w`.

File: coalib/testing/LocalBearTestHelper.py [hunk 1]
```diff
@@ -193,6 +193,9 @@ def check_results(self,
         :param lines:            The lines to check. (List of strings)
         :param results:          The expected list of results.
         :param filename:         The filename, if it matters.
+        :param check_order:      Whether to check that the elements of results
+                                 (expected list) and that of the list generated
```
Comment (Makman2) [inline coalib/testing/LocalBearTestHelper.py]: > (expected list)

? Would remove that.

File: coalib/testing/LocalBearTestHelper.py [hunk 2]
```diff
@@ -193,6 +193,9 @@ def check_results(self,
         :param lines:            The lines to check. (List of strings)
         :param results:          The expected list of results.
         :param filename:         The filename, if it matters.
+        :param check_order:      Whether to check that the elements of results
```
Comment (Makman2) [inline coalib/testing/LocalBearTestHelper.py]: If you refer to the parameter (and I think you do), you should put `results` into double-backticks :+1:

File: coalib/testing/LocalBearTestHelper.py [hunk 3]
```diff
@@ -193,6 +193,9 @@ def check_results(self,
...
+                                 (expected list) and that of the list generated
+                                 from get_results are in the same order or not.
```
Comment (Makman2) [inline coalib/testing/LocalBearTestHelper.py]: That `get_results` is used is an implementation detail and I wouldn't mention it explicitly.

**Comment by Makman2** [comment]
unack 353ccae2fed1be550ddf2b6b1bcb042c523ef92f

**Comment by pareksha** [comment]
@Makman2 made the changes. Please review

**Comment by Makman2** [comment]
ack dccd6af6390823e15aef6303cdeaaab1feae016d

**Comment by Makman2** [comment]
@gitmate-bot rebase

**Comment by Makman2** [comment]
@gitmate-bot ff

**Pull request merged**